addons/binding/org.openhab.binding.konnected/src/main/java/org/openhab/binding/konnected/internal/KonnectedHTTPUtils.java

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright (c) 2010-2018 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.konnected.internal;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.Properties;
+import java.util.concurrent.TimeUnit;
+
+import org.eclipse.smarthome.io.net.http.HttpUtil;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * HTTP Get and Put reqeust class.
+ *
+ * @author Zachary Christiansen - Initial contribution
+ */
+public class KonnectedHTTPUtils {
+    private final Logger logger = LoggerFactory.getLogger(KonnectedHTTPUtils.class);
+    private static final int REQUEST_TIMEOUT = (int) TimeUnit.SECONDS.toMillis(30);
+    private String logTest = "";
+
+    public KonnectedHTTPUtils() {
```

Comment:
This empty constructor is not needed, if you leave it out the end result is the same.

Reply:
removed